python/neuroglancer/chunks.py

```diff
@@ -44,6 +44,5 @@ def encode_jpeg(subvol):
     img.save(f, "JPEG")
     return f.getvalue()
 
-
 def encode_raw(subvol):
-    return subvol.tostring('C')
\ No newline at end of file
+    return subvol.tostring('F')
```

Comment:
ok! I might need to add a transpose to my code in superdownsample tho maybe we can add a flag. I have a feeling 'F' is more expensive than 'C' tho the cost is probably quite low.